deployments/chef/metadata.rb

```diff
@@ -17,5 +17,7 @@
 
 depends 'windows', '>= 4.3.4'
 
+gem 'rubyzip', '< 2.0.0'
```

Comment:
I'm a bit wary of how this will interact with people's environments, but I guess we can try it as long as we make it a separate release that people can roll back if problematic.

Reply:
> I'm a bit wary of how this will interact with people's environments, but I guess we can try it as long as we make it a separate release that people can roll back if problematic.

I had the same concerns, but the chef-client should be able to handle multiple versions of the same gem if necessary.